.golangci.yml

```diff
@@ -0,0 +1,150 @@
+---
```

Comment:
This is a clone of https://github.com/ossf/scorecard/blob/873308016c73b99b728e4609b23c434b28f32f6a/.goreleaser.yml#L1